cpc/cpc_sketch.go

```diff
@@ -513,3 +535,149 @@ func (c *CpcSketch) ToCompactSlice() ([]byte, error) {
     return (byte[]) wmem.getArray();
   }
 */
+
+// GetFamily returns the CPC family identifier.
+func (c *CpcSketch) GetFamily() internal.Family {
```

Comment:
We can't expose internal package.

for now, just remove that GetFamily - we ll need to find a way to expose that across all sketches properly